### ISSUE TITLE
Rubberduck can not be sucessfully installed for single user if once been installed for All Users

### ISSUE THREAD

**Issue by AstrocalcVB** [rubberduck-vba/Rubberduck#5785]
**Rubberduck version information**
	Rubberduck version 2.5.2.5927 loading:
	Operating System: Microsoft Windows NT 6.2.9200.0 x64
	Host Product: Visual Basic x86
	Host Version: 6.00.9782
	Host Executable: VB6.EXE;

**Description**
I don't know if this is by design but I had pre-2.5.2.5920 installed for All users and before installing .5927 I uninstall the old one, with my uninstall cleanup tool also removing the folders in "Appdata\Roaming". On launch of VB6 I got the message that "Rubberduck could not be loaded. Remove it from list of avaliable Add-Ins? Yes/No

After uninstall of .5927 (no folder were available to remove, i.e. they were never created) and install again, this time again for All users it loads as expected and folders are created.

**To Reproduce**
1. Install 2.5.2.5920 for "All users"
2. Uninstall 2.5.2.5920 also removing folders in Appdata\Roaming
3. Install 2.5.2.5927 for single user
4. Start VB6

**Expected behavior**
Rubberduck is expected to load with VB6, but doesn't.

**Logfile**
No logfile was created as no Log folder was created.
**Additional context**
I suspect that the problem may arise from that the Rubberduck folder never was created in "Appdata\Roaming". I user a 3rd party uninstall tool, which also scans registry for leftover items as well as folder items (if it got them from registry or not I don't know). Apparently the RD uninstall leaves these folder behind if once installed, and for some reason it expected or seemed to believe they where there and didn't notice they were missing. All my assumptions though.

**Comment by bclothier** [comment]
The matter of leaving behind folder is a tricky one. We do not delete that folder because it contains user's customizations and it would be inconvenient to lose it if it was uninstalled then later installed (particularly in response to correct some kind of install or registration issues). 

As a consequence, the installer will delete the folder. 

However, the VB6 not loading it might be actually a different problem. Can you please confirm if you are running VB6 as an administrator? That would not be a scenario we can support --- running per-user Rubberduck on a elevated instance of VB6 (or even Office).  Microsoft has some security around mixing the registry between HCKU and HKLM which prevents us from running per-user add-ins in an elevated context and in this case the only fix is to install it for all users.

**Comment by mansellan** [comment]
It is theoretically possible to run VB6 in a non-elevated context, but it causes many problems as it lacks rights to register objects for debugging. I think at the very least it's impossible to debug ActiveX projects, but I can't recall for standard exes.

Are you trying to run VB6 non-elevated?

**Comment by AstrocalcVB** [comment]
Yes, I am running Win10 and VB6 as Administrator, although not with the originally created "Administrator" account. Not a big deal for me as I got it installed but thought I would report it. However, I think it would be nice with a notice about this, either in the installer or the Installation documentation (if not already, I haven't checked).

**Comment by retailcoder** [comment]
@AstrocalcVB FYI the release notes of the "latest release" can be found at https://github.com/rubberduck-vba/Rubberduck/releases/latest - pre-release builds don't get this information (for now anyway), but it's the same content every release, copied here for convenience:

---

[**Installation instructions**](https://github.com/rubberduck-vba/Rubberduck/wiki/Installing)

**VBA6 | VBA7 x86/x64**

Installs without administrator privileges for the current Windows user login.
Local admin users should install for all users and register the add-in for their login at the end of the installation - add-in will need to be manually registered for other users on the same machine.

**VB6**

If Visual Studio 6.0 is executed with administrator privileges, Rubberduck should be installed for all users.

---
